IFC 4.3/Align_1/UT_Align_1.IFC

```diff
@@ -0,0 +1,103 @@
+/* Generated by TcpMDT - V8.5
+   by APLITOP, S.L. (www.aplitop.com)
+*/
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('UT_Align_1.IFC','2020-11-16T12:56:17',(),(),'IfcOpenShell 0.5.0-dev','IfcOpenShell 0.5.0-dev','');
+FILE_SCHEMA(('IFC4X3_RC2'));
+ENDSEC;
+DATA;
+#1=IFCACTORROLE(.USERDEFINED.,'',$);
+#2=IFCADDRESS(.HOME.,'',$);
+#3=IFCPERSON('Javier','','',$,$,$,(#1),(#2));
+#4=IFCADDRESS(.OFFICE.,'',$);
+#5=IFCORGANIZATION('','Aplitop','',$,(#4));
+#6=IFCPOSTALADDRESS(.OFFICE.,$,$,'',(''),$,'','','','');
+#7=IFCAPPLICATION(#5,'8.5','TCP-Digital Terrain Model','TcpMDT');
+#8=IFCPERSONANDORGANIZATION(#3,#5,$);
+#9=IFCOWNERHISTORY(#8,#7,$,$,$,$,$,0);
+#10=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#11=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#12=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#14=IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453293),#12);
+#15=IFCCONVERSIONBASEDUNIT(#10,.PLANEANGLEUNIT.,'Degrees',#14);
+#16=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#17=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#18=IFCUNITASSIGNMENT((#11,#15,#16,#17));
+
+/* Project */
+#19=IFCPROJECT('3FBN6HavH7jgjsr54toxUp',$,'UT_Align_1','',$,'',$,(),#18);
+
+/* Site */
+#20=IFCDIRECTION((1.,0.,0.));
+#21=IFCDIRECTION((0.,0.,1.));
+#22=IFCCARTESIANPOINT((0.,0.,0.));
+#23=IFCAXIS2PLACEMENT3D(#22,#21,#20);
+#24=IFCGEOMETRICREPRESENTATIONCONTEXT('Body','Model',3,$,#23,$);
+#25=IFCDIRECTION((1.,0.,0.));
+#26=IFCDIRECTION((0.,0.,1.));
+#27=IFCCARTESIANPOINT((0.,0.,0.));
+#28=IFCAXIS2PLACEMENT3D(#27,#26,#25);
+#29=IFCLOCALPLACEMENT($,#28);
+#30=IFCSITE('27GNfc9M96KOY3cASzR0Sw',#9,'None',$,$,#29,$,$,.ELEMENT.,$,$,$,$,#6);
+
+/* Site in Project */
+#31=IFCRELAGGREGATES('0rv$BufPD0sAessPuJZe_O',#9,$,$,#19,(#30));
+
+/* Empty road */
+#32=IFCROAD('1FXfkfy6b93BwiEai_gN$E',$,'None',$,'IfcRoad',$,$,$,.COMPLEX.);
+
+/* Road in project */
+#33=IFCRELAGGREGATES('2uInlzG_X1_glgN9RNs$c8',#9,$,$,#30,(#32));
+
+/* Alignment: horizontal and vertical alignment */
+#34=IFCALIGNMENT('0XTxY6KDrCdQRFzSOb6jwJ',$,'UT_Align_1',$,$,$,$,#35,$);
+#35=IFCALIGNMENTCURVE(#36,#73,$);
+
+/* Horizontal alignment */
+#36=IFCALIGNMENTHORIZONTAL('06__Wngff6zAUzQvB4Fnt4',$,$,$,$,$,$,0.,(#60,#38,#40,#42,#44,#46,#62,#48,#50,#52,#64,#54,#56,#58,#66));
+#37=IFCCARTESIANPOINT((335095.95046541,4084593.74863239));
+#38=IFCALIGNMENTHORIZONTALSEGMENT($,$,#37,357.802095555,-25.,-25.,39.84063746,$,.CIRCULARARC.);
```

Comment:
Negative radius of curvature indicates right hand turning segment, where as image and next segment start point indicates left hand turn.  I believe all radius values in the sample should be multiplied by negative 1.

Reply:
Fixed in 1954276